doc/source/whatsnew/v0.18.1.rst

```diff
@@ -190,6 +190,7 @@ Partial string indexing now matches on ``DateTimeIndex`` when part of a ``MultiI
 On other levels
 
 .. ipython:: python
+   :okwarning:
```

Comment:
Could you just change this to use `axis_ops.swaplevel`?

Reply:
i dont think we'd want to change this in the old doc

Reply:
Okay could we use `code block` instead?